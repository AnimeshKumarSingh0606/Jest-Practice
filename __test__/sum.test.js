import sum from '../src/sum';

describe('testing sum function', () => {
  
  test('1 + 2 is equal to 3', () => expect(sum(1, 2)).toBe(3));
  
  test('2 + 2 is equal to 4', () => expect(sum(2, 2)).toMatchSnapshot());
});
