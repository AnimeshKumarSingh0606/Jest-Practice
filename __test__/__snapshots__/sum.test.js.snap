

exports[`testing sum function 2 + 2 is equal to 4 1`] = `4`;
